pkgs/development/cuda-modules/generic-builders/manifest.nix

```diff
@@ -243,16 +253,14 @@ backendStdenv.mkDerivation (
     meta = {
       description = "${redistribRelease.name}. By downloading and using the packages you accept the terms and conditions of the ${finalAttrs.meta.license.shortName}";
       sourceProvenance = [sourceTypes.binaryNativeCode];
-      platforms =
-        lists.concatMap
-          (
-            redistArch:
-            let
-              nixSystem = builtins.tryEval (flags.getNixSystem redistArch);
-            in
-            if nixSystem.success then [nixSystem.value] else []
-          )
-          supportedRedistArchs;
+      platforms = trivial.pipe supportedRedistArchs [
+        # Map each redist arch to the equivalent nix system or null if there is no equivalent.
+        (trivial.flip attrsets.genAttrs flags.getNixSystem)
+        # Filter out entries which do not have a corresponding nix system
+        (attrsets.filterAttrs (_: value: value != null))
+        # Get the remaining nix systems
+        attrsets.attrValues
+      ];
```

Comment:
Nice trick. Looks very clean.

Reply:
Naming things is hard so I prefer to use functions where possible :)